writerService.go

```diff
@@ -1,21 +1,79 @@
 package main
 
 import (
-	"fmt"
-	"io"
+	"github.com/influxdata/influxdb1-client/v2"
+	"log"
+	"strconv"
+	"time"
 )
 
-type MeasurementsWriterService struct{}
+type MeasurementsWriterService struct {
+	db           string
+	writerClient client.Client
+}
+
+func NewMeasurementsWriterService(dbAddress string) *MeasurementsWriterService {
+	clt, err := client.NewHTTPClient(client.HTTPConfig{
+		Addr: dbAddress,
+	})
+	if err != nil {
+		log.Panic(err)
```

Comment:
This panic needs clearer message. My suggestion: `log.Panicf("could not initialize influx connection: %s", err.Error())`